Src/WitsmlExplorer.Api/Services/LogDataReader.cs

```diff
@@ -63,23 +69,36 @@ public LogDataReader(IWitsmlClient witsmlClient, WitsmlLog sourceLog, List<strin
                     mnemonics.Insert(0, indexMnemonic);
                 }
             }
+            Produce();
         }
 
-        /// <summary>
-        /// Fetches and returns the next batch of log data.
-        /// </summary>
-        /// <returns>WitsmlLogData or, if there is no more data to fetch, null</returns>
-        public async Task<WitsmlLogData> GetNextBatch()
+        private async void Produce()
```

Comment:
[Best practice](https://learn.microsoft.com/en-us/archive/msdn-magazine/2013/march/async-await-best-practices-in-asynchronous-programming) is to prefer Task as return type instead of void. Is there a reason to use void here?

(Note that the call to `Produce();` above needs to be wrapped in a `Task.Run(Produce)` if the return type is changed.

Reply:
No reason to use void. Replaced with Task.Run